### DISCUSSION TITLE
cpptools very slow for large project

### DISCUSSION THREAD

**Discussion by piterzhang** [microsoft/vscode-cpptools#10397] [Ideas]
1.when I use cpptool to read linux code,the speed of go to function define is very slowly and some times can't work 
2. then i try gun global in vscode to read linux code ， surprised ,very fast. but used gun global cant used cpptool,I need cpptool to debug also

Q:

1.can give user a selectc used cpptool or gun global
 
2.if cpptool can work like gun global, generate data base before read code
let it fast when read large project

**Comment by sean-mcmanus** [comment]
If you want to use another extension for language service processing and cpptools for debuggng you can set C_Cpp.intelliSenseEngine to "disabled".

For the Go to Definition speed, it could be caused by different issues. You could open an issue https://github.com/microsoft/vscode-cpptools/issues if you want us to investigate the performance further.
